src/oid.rs

```diff
@@ -14,6 +14,7 @@ lazy_static! {
     // PKCS #1
     pub static ref rsaEncryption: ObjectIdentifier = vec![1, 2, 840, 113549, 1, 1, 1].into();
     pub static ref sha256WithRSAEncryption: ObjectIdentifier = vec![1, 2, 840, 113549, 1, 1, 11].into();
+    pub static ref sha256WithECDSAEncryption: ObjectIdentifier = vec![1, 2, 840, 10045, 4, 3, 2].into();
```

Comment:
Where did you get this name from?

Reply:
Follows same naming convention as: 'sha256WithRSAEncryption'

Actual value from: http://oid-info.com/cgi-bin/display?oid=1.2.840.10045.4.3.2&action=display
And from looking at other certificates.

Reply:
It's not a convention. The name is defined in the appropriate RFC, 3447 for RSA, 5480 for ECDSA.